janus_core/janus_types.py

```diff
@@ -56,6 +58,18 @@ class ASEOptRunArgs(TypedDict, total=False):
     steps: int
 
 
+class LogLevel(Enum):
+    """Supported options for logger levels."""
+
+    DEBUG = logging.DEBUG
+    INFO = logging.INFO
+    WARNING = logging.WARNING
+    ERROR = logging.ERROR
+
+
+LogLevelType = Literal[LogLevel]
```

Comment:
Not sure this is right, [this says](https://stackoverflow.com/questions/52624736/type-annotations-for-enum-attribute) you should just be able to use the `Enum` itself. As written `LogLevelType` asks you to pass the LogLevel class itself, not any of its members

Reply:
Ah sorry, I misinterpreted your previous suggestion.

Yes, based on https://typing.readthedocs.io/en/latest/spec/literal.html#literal-types, just using LogLevel as the type hint should be "approximately" equivalent to what I had before (`Literal[LogLevel.debug, ...]`, so hopefully it's ok now.